lib/workers/repository/update/pr/index.spec.ts

```diff
@@ -473,9 +473,6 @@ describe('workers/repository/update/pr/index', () => {
           sourceBranch: 'renovate/dummy-1.x',
         },
       ]);
-      expect(platform.createPr.mock.calls[0][0].prBody).toContain(
-        'this Pin PR'
```

Comment:
Revert this removal and instead change the text to match something in the new string